swan-lake/featured-scenarios/work-with-data-using-queries-in-ballerina.md

```diff
@@ -426,31 +426,35 @@ Countries with highest deaths:[["India",980976],["USA",880976],["Afghanistan",73
 Countries with number of Recovered patients:[["Afghanistan",146084],["USA",43892277],["India",33892279]]
 Found erroneous entries for countries: ["Sri Lanka","India"]
 ```
-## Group COVID-19 data by continent
 
-### Create the `groupDeathsByContinent` function
+## Aggregating
+
+### Create the `getTotalNumberOfDeaths` function
+
+In order to aggregate the input records of query we can use `collect` clause. If we use `collect` clause, all the variables in the query expression become aggregated variables. We can use aggregated variables as an argument in the function call or single element list constructor. 
```

Comment:
I think we need to say the aggregated values can be arguments to rest params of langlib functions

Reply:
It is mentioned in the next paragraph. Isn't that enough?

Reply:
Then lets remove this part

Reply:
Changed it, please have a look.